Competition2017/src/competition/subsystems/fuel_launcher/commands/MaintainShooterPowerCommand.java

```diff
@@ -0,0 +1,25 @@
+package competition.subsystems.fuel_launcher.commands;
+
+import competition.subsystems.shooter.SideShooterSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class MaintainShooterPowerCommand extends BaseCommand {
+
+    final SideShooterSubsystem sideShooter;
+
+    public MaintainShooterPowerCommand(SideShooterSubsystem sideShooter) {
+        this.sideShooter = sideShooter;
+        this.requires(this.sideShooter);
+    }
+
+    @Override
+    public void initialize() {
+    
+    }
+    
+    @Override
+    public void execute() {
+        sideShooter.updateTelemetry();
```

Comment:
All this does is update telemetry? If so, `MaintainShooterPowerCommand` is a misnomer; it's just a no-op WRT shooter power.